crates/bevy_render/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub mod extract_resource;
 pub mod globals;
 pub mod gpu_component_array_buffer;
 pub mod mesh;
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Is there a reason why the entire module is hidden on wasm32? Otherwise we need to document this or this might be a sharp API edge.

Reply:
The pipelined rendering plugin already doesn't work on wasm32. So I don't think removing it is a significant change. Though I agree that it should be documented.